Assets/MRTK/SDK/Profiles/DefaultMixedRealityInputSimulationProfile.asset

```diff
@@ -41,7 +41,7 @@ MonoBehaviour:
   moveUpDown: UpDown
   lookHorizontal: AXIS_4
   lookVertical: AXIS_5
-  simulateEyePosition: 0
+  defaultEyeGazeSimulationMode: 1
```

Comment:
@julenka I think this enables eye gaze = camera by default and I know that we've had plenty of conversations in the past about why this isn't good but I never remember why @keveleigh to keep me honest if I'm making this up

Reply:
didn't mean to change the default reverting this change.